src/devices/src/virtio/net/device.rs

```diff
@@ -730,6 +730,12 @@ impl Net {
             }
         }
     }
+
+    /// Process device virtio queue(s).
+    pub fn process_virtio_queues(&mut self) {
```

Comment:
Why no process_virtio_queues() in Vsock ?

Reply:
Vsock state machine is not resilient to _control packets_ loss. Epoll events loss is just a small part of the story.

I've opened #2218 to track and discuss this in detail.

I guess for know, we can have it as a _known issue_ with a clear workaround:
> close all vsock connections prior to snapshotting